floss/identify.py

```diff
@@ -151,8 +152,7 @@ def find_decoding_function_features(vw, functions, disable_progress=False) -> Tu
 
             f = viv_utils.Function(vw, function_address)
 
-            function_data = {"meta": get_function_meta(f), "features": list()}
-
+            function_data = {"meta": get_function_meta(f), "features": [] , "call_count":len(list(vw.getXrefsTo(function_address)))}
```

Comment:
I think `xrefs_to` is more accurate than `call_count` and similarly throughout here.